src/engine/audio/Soundtrack.hxx

```diff
@@ -53,7 +63,7 @@ struct Soundtrack
    */
   bool isPlayable : 1;
 
-#ifdef USE_OPENAL_SOFT
+#ifdef USE_AUDIO
```

Comment:
This file is also not added when USE_AUDIO is false. It should be safe to remove the ifdefs